generative_data_prep/data_prep/data_prep.py

```diff
@@ -35,7 +35,9 @@ def data_prep_main(silent: bool, tokenizer: PreTrainedTokenizerBase,
                    input_packing_config: PackingConfig,
                    packing_boundary: BoundaryType,
                    attention_boundary: BoundaryType,
-                   disable_space_separator: bool, prompt_keyword: str,
+                   disable_space_separator: bool, 
+                   keep_prompt_only_sequences: bool,
+                   prompt_keyword: str,
```

Comment:
could you add documentation for `prompt_keyword`

Reply:
I think he means keep_prompt_only_sequences, but +1

Reply:
Actually, we don't have documentation for prompt_keyword. I'll add it.